src/App.js

```diff
@@ -1,9 +1,44 @@
 import "./App.css";
 
+import { Routes, Route } from "react-router-dom";
+
+import {
+  Navbar,
+  Playlist,
+  Videos,
+  Sidebar,
+  ChannelPage,
+  VideoPlayer,
+  SignIn,
+  SignUp,
+  Subscribed,
+  LikedVideos,
+} from "./Components";
+
 function App() {
   return (
     <div className="App">
-      <h1>React</h1>
+      <Navbar />
+      <div className="routesDiv">
+        <Routes>
```

Comment:
You could try to extract these routes out into another component and have that component along with navbar and sidebar in the app.js file